UPGRADE.md

```diff
@@ -30,6 +30,11 @@ However, several replacements have been put in place depending on the `$queryPar
 The usage of `QueryBuilder::getQueryPart()` and `::getQueryParts()` is deprecated. The query parts
 are implementation details and should not be relied upon.
 
+## Deprecated `AbstractMySQLPlatform::getColumnTypeSQLSnippets()`
+
+Using `AbstractMySQLPlatform::getColumnTypeSQLSnippets()` has been deprecated.
+Use `AbstractMySQLPlatform::getColumnTypeSQLSnippet()` instead.
+
```

Comment:
Sorry, I should've mentioned that we add new messages at the top of the file. 😓

Reply:
Fixed.